search/populate.py

```diff
@@ -0,0 +1,52 @@
+from io import BytesIO
+
+from flask import Blueprint, Response
+from flask import current_app as app
+
+import requests
+from elasticsearch import Elasticsearch
+from lxml import etree
+
+from search.utils import strip_slashes
+
+
+def get_populate_blueprint() -> Blueprint:
+    blueprint = Blueprint('populate', __name__)
+
+    @blueprint.route('/populate', methods=['POST'])
+    def populate() -> Response:
+        """
+        Route for triggering indexing of known paths. Will be used as a proof of
+        concept. This should be removed before MVP release.
```

Comment:
We will need a way to index known paths long term (eg for bringing up a search service later on/reindexing for some reason). Might be worth extract that code away from the temporary part (ie the HTTP endpoint).

Reply:
Agreed. This should be extracted into a command that can be called from anywhere. I would prefer to have search better defined before doing so as I expect to change a lot of the code in the PR, which in turn will generate a lot of discussion that I would rather avoid for this PR.